core/drivers/clk/sam/at91_clk.h

```diff
@@ -186,6 +186,14 @@ at91_clk_register_master_div(struct pmc_data *pmc,
 			     const struct clk_master_layout *layout,
 			     const struct clk_master_charac *charac);
 
+struct clk *at91_clk_sama7g5_register_master(struct pmc_data *pmc,
```

Comment:
It suggest that you state somewhere that both `parent` and `mux_table` have `num_parents` cells.

Reply:
`mux_table_size` added and assigned equal to `num_parents` in `at91_clk_register_generated()`.

Reply:
An inline description comment could clarify that when `mux_table` is not NULL it shall hold `num_parents` cells. This is only a suggestion.